client/my-sites/email/email-management/home/mailpoet-upsell.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button, Card } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import mailpoetSymbol from 'calypso/assets/images/email-providers/mailpoet-symbol.svg';
+import SectionHeader from 'calypso/components/section-header';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+
+export default function MailPoetUpsell() {
+	const translate = useTranslate();
+	const selectedSite = useSelector( getSelectedSite );
+
+	return (
+		<div className="mailpoet-upsell">
+			<SectionHeader label={ translate( 'Newsletters' ) } />
+
+			<Card>
+				<div className="mailpoet-upsell__icon">
+					<img src={ mailpoetSymbol } alt="MailPoet" />
```

Comment:
Maybe the alternative text could be more specific?

```suggestion
					<img src={ mailpoetSymbol } alt="MailPoet logo" />
```

Reply:
Sure! I suppose that's more in line with the symbols we show for Google/WordPress.